ui/pages/settings/flask/snaps-list-tab/view-snap/index.scss

```
@@ -0,0 +1,84 @@
+div .view-snap-subheader {
+  @media screen and (max-width: $break-small) {
+    display: flex;
+    margin-right: 0;
+    padding: 0 0 16px;
+    flex-direction: column;
+    align-items: center;
+    gap: 8px;
+    height: max-content;
+  }
+}
+
+.view-snap-title {
+  @media screen and (max-width: $break-small) {
+    padding-bottom: 16px;
+  }
+}
+
+.view-snap-pill-toggle-container {
+  align-items: center;
+  display: flex;
+  flex-grow: 1;
+
+  @media screen and (max-width: $break-small) {
+    width: 100%;
+    justify-content: space-between;
+  }
+}
+
+.view-snap-content-container {
+  @media screen and (max-width: $break-small) {
+    width: 100%;
+  }
+}
+
+.view-snap-pill-container {
+  @media screen and (max-width: $break-small) {
+    padding-left: 0;
+    display: inline-block;
+  }
+}
+
+.view-snap-toggle-container {
+  @media screen and (max-width: $break-small) {
+    padding-left: 0;
+    margin-left: auto;
+    display: inline-block;
+  }
+}
+
+.settings-page__content-item.view-snap-section {
+  border-bottom: 1px solid #d8d8d8;
+  padding-bottom: 18px;
+  margin-bottom: 16px;
+}
+
+.view-snap-section.snap-permission-list {
+  padding-bottom: 0;
+}
+
+.settings-page__content-item .permissions-connect-permission-list .permission {
+  padding-top: 16px;
+
+  &:last-child {
+    border-bottom: none;
+  }
+}
+
+.view-snap__remove__button {
+  @media screen and (max-width: $break-small) {
+    align-self: center;
+  }
+}
+
+.view-snap-section {
+  .connected-sites-list__content-row {
+    border-top: none;
+    border-bottom: 1px solid #d6d9dc;
```

There is a variable for this color:
```suggestion
    border-bottom: 1px solid $ui-2;
```

---

Will change, thanks!